api/admin/routes.py

```diff
@@ -63,6 +63,15 @@ def admin_sign_in():
 def work_details(data_source, identifier):
     return app.manager.admin_work_controller.details(data_source, identifier)
 
+@app.route('/admin/works/<data_source>/<identifier>/classifications', methods=['GET'])
```

Comment:
This only returns genre classifications so maybe it should be called genres? Or do you want to expand it later?